checker/build.gradle

```diff
@@ -21,7 +21,11 @@ sourceSets {
             exclude '**/org/checkerframework/checker/resourceleak/MustCallInference.java'
         }
     }
-    testannotations
+    testannotations {
+        java {
```

Comment:
How did this work before? Is this the default made explicit?

Reply:
TBH, I am not sure what is the default in this situation. I find make it explict should allow us to have more control what we include?